manifests/filter/open_tracing.pp

```diff
@@ -0,0 +1,170 @@
+# == Resource:  repose::filter::open_tracing
+#
+# This is a resource for enabling emitting traces to an OpenTracing tracer (current supports Jaeger).
+# Only usable in >= 8.8.3.0
+#
+# === Parameters
+#
+# [*ensure*]
+# Bool. Ensure config file present/absent
+# Defaults to <tt>present</tt>
+#
+# [*filename*]
+# String. Config filename.
+# Defaults to <tt>open-tracing.cfg.xml</tt>
+#
+# [*service_name*]
+# String. Application name
+# Defaults to <tt>repose</tt>
+#
+# [*http_connection_endpoint*]
+# String. Set to http endpoint to push traces to.
+# Defaults to <tt>http://localhost:12682/api/traces</tt>
+#
+# [*http_connection_username*]
+# String. Set if http endpoint requires username/password authentication.  Must be used in conjunction with password
+# Defaults to <tt>undef</tt>
+#
+# [*http_connection_password*]
+# String. Set if http endpoint requires username/password authentication.  Must be used in conjunction with username
+# Defaults to <tt>undef</tt>
+#
+# [*http_connection_token*]
+# String. Set if http endpoint requires token authentication.
+# Defaults to <tt>undef</tt>
+#
+# [*udp_connection_port*]
+# Integer. Set to port the agent is listening on.  Used in tandem with udp_connection_host.
+# Defaults to <tt>undef</tt>
+#
+# [*udp_connection_host*]
+# String. Set to host the agent is listening on.  Used in tandem with udp_connection_port.
+# Defaults to <tt>undef</tt>
+#
+# [*constant_toggle*]
```

Comment:
I feel less strongly about this one, but it does have the same problem.

Reply:
Addressed by changing to `undef`